charts/gateway-helm/templates/certgen-rbac.yaml

```diff
@@ -110,6 +114,7 @@ metadata:
   annotations:
     "helm.sh/hook": pre-install, pre-upgrade
     "helm.sh/hook-weight": "-1"
+    "argocd.argoproj.io/sync-wave": "-1"  # Ensure rbac is created before the certgen job when using ArgoCD.
```

Comment:
instead, can you apply hook-weigh: -1 to the other certgen rbac ?

Reply:
Does Argo CD honor the "helm.sh/hook-weight" annotation?

Reply:
Oh NVM, it does.